packit/exceptions.py

```diff
@@ -4,3 +4,7 @@ class PackitException(Exception):
 
 class PackitConfigException(PackitException):
     pass
+
+
+class PackitFedpkgException(Exception):
```

Comment:
why not inherit from PackitException?

Reply:
PackitFedpkgException is deleted. It is now obsolete